api/buffer/design.go

```diff
@@ -361,35 +364,43 @@ var Export = Type("Export", func() {
 	Description("Represents a mapping from imported data to a destination table.")
 	exportId("May be omitted, in which case it will be generated from the name. The value cannot be changed later.")
 	name("export", "GitHub Changed Files")
+	Attribute("mapping", Mapping, func() {
+		Description("Export column mapping.")
+	})
+	Attribute("conditions", ImportConditions, func() {
+		Description("Table import conditions.")
+	})
+	Required("name", "mapping")
+})
+
+var Mapping = Type("Mapping", func() {
+	Description("Export column mapping.")
 	Attribute("tableId", String, func() {
 		Description("Destination table ID.")
 	})
 	Attribute("incremental", Boolean, func() {
 		Description("Enables incremental loading to the table.")
 		Default(true)
 	})
-	Attribute("columns", ArrayOf(ColumnMapping), func() {
+	Attribute("columns", ArrayOf(Column), func() {
 		Description("List of export column mappings. An export may have a maximum of 50 columns.")
 	})
-	Attribute("conditions", ImportConditions, func() {
-		Description("Table import conditions.")
-	})
-	Required("name", "tableId", "columns")
+	Required("tableId", "columns")
 })
 
-var ColumnMapping = Type("ColumnMapping", func() {
+var Column = Type("Column", func() {
```

Comment:
Tady je pořád problém v tom, že ty columns nejsou polymorphic i v payloadu. Převést to na ty polymorphic columns jde, ale není to nejlepší UX.

Reply:
Zatial to este asi nechajme ak to funguje, 
trocha som to pozeral, zatial mam take nieco:
```
Meta(oneof.Meta, json.MustEncodeString([]*openapi.Schema{
	{
		Description: "ID column",
		Type:        openapi.Object,
		Required:    []string{"type"},
		Properties: map[string]*openapi.Schema{
			"type": {Type: openapi.String, Enum: []any{column.IDType}},
		}},
	{
		Description: "Datetime column",
		Type:        openapi.Object,
		Required:    []string{"type"},
		Properties: map[string]*openapi.Schema{
			"type": {Type: openapi.String, Enum: []any{column.DatetimeType}},
		}},
	{
		Description: "IP column",
		Type:        openapi.Object,
		Required:    []string{"type"},
		Properties: map[string]*openapi.Schema{
			"type": {Type: openapi.String, Enum: []any{column.IPType}},
		}},
	{
		Description: "Body column",
		Type:        openapi.Object,
		Required:    []string{"type"},
		Properties: map[string]*openapi.Schema{
			"type": {Type: openapi.String, Enum: []any{column.BodyType}},
		}},
	{
		Description: "Headers column",
		Type:        openapi.Object,
		Required:    []string{"type"},
		Properties: map[string]*openapi.Schema{
			"type": {Type: openapi.String, Enum: []any{column.HeadersType}},
		}},
	{
		Description: "Templated column",
		Type:        openapi.Object,
		Required:    []string{"type", "language", "undefinedValueStrategy", "content", "dataType"},
		Properties: map[string]*openapi.Schema{
			"type":                   {Type: openapi.String, Enum: []any{column.TemplateType}},
			"language":               {Type: openapi.String, Enum: column.TemplateLanguageValues()},
			"undefinedValueStrategy": {Type: openapi.String, Enum: column.UndefinedValueStrategyValues()},
			"content":                {Type: openapi.String},
			"dataType":               {Type: openapi.String, Enum: column.DataTypeValues()},
		}},
}, false))
```

Co trocha vylepsi openapi schemu, este chcem na to pozriet.